tlsn/tlsn-core/src/merkle.rs

```diff
@@ -144,7 +153,17 @@ impl MerkleTree {
     }
 
     /// Creates an inclusion proof for the given `indices`
+    ///
+    /// # Panics
+    ///
+    /// - if `indices` is not sorted.
```

Comment:
@sinui0 , instead of imposing the burden on the caller to sort the indices, we can just sort them right in this fn.